<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSettingsRegister</class>
 <widget class="QWidget" name="KSettingsRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register settings</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string comment="@title The way in which items will be displayed">Display</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="kcfg_ShowGrid">
         <property name="text">
          <string>Show a grid in the register</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_ShowRegisterDetailed">
         <property name="text">
          <string>Show all register entries in full detail</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_LedgerLens">
         <property name="whatsThis">
          <string>Using the ledger lens shows the details for the transaction that has focus in the ledger. Usually, when using the transaction form, only a one line summary is displayed for each transaction as the details are shown in the form.</string>
         </property>
         <property name="text">
          <string>Use the ledger lens</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_TransactionForm">
         <property name="text">
          <string>Show transaction form</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_AlwaysShowNrField">
         <property name="text">
          <string>Always show a No. field</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_ShowFancyMarker">
         <property name="whatsThis">
          <string>Draws a larger header above each group of transactions. The grouping depends on the current sort order.</string>
         </property>
         <property name="text">
          <string>Show group header between transactions</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_ShowFiscalMarker">
         <property name="text">
          <string>Show header for the previous and current fiscal year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_ShowPlannedScheduleDates">
         <property name="whatsThis">
          <string>If a planned transaction is overdue, today's date is used by default as the posting date. With this option, the originally planned date is used instead.</string>
         </property>
         <property name="text">
          <string>Display overdue schedules with planned date</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sorting">
      <attribute name="title">
       <string>Sorting</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget3">
         <widget class="QWidget" name="tab3">
          <attribute name="title">
           <string>Normal view</string>
          </attribute>
          <layout class="QHBoxLayout">
           <item>
            <widget class="KLineEdit" name="kcfg_sortNormalView"/>
           </item>
           <item>
            <widget class="TransactionSortOption" name="m_sortNormalView" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab4">
          <attribute name="title">
           <string>Reconciliation view</string>
          </attribute>
          <layout class="QHBoxLayout">
           <item>
            <widget class="KLineEdit" name="kcfg_sortReconcileView"/>
           </item>
           <item>
            <widget class="TransactionSortOption" name="m_sortReconcileView" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="TabPage">
          <attribute name="title">
           <string>Search view</string>
          </attribute>
          <layout class="QHBoxLayout">
           <item>
            <widget class="KLineEdit" name="kcfg_sortSearchView"/>
           </item>
           <item>
            <widget class="TransactionSortOption" name="m_sortSearchView" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="qLabel2">
         <property name="text">
          <string>Use the &lt;i&gt;left&lt;/i&gt; and &lt;i&gt;right&lt;/i&gt; buttons to add and remove sort options. Use the &lt;i&gt;up&lt;/i&gt; and &lt;i&gt;down&lt;/i&gt; buttons to modify the sort order. Double-Click a selected entry to toggle the sort order between &lt;i&gt;ascending&lt;/i&gt; and &lt;i&gt;descending&lt;/i&gt;.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage1">
      <attribute name="title">
       <string>Data entry</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="kcfg_AutoIncCheckNumber">
         <property name="text">
          <string>Auto increment check number</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_FocusChangeIsEnter">
         <property name="text">
          <string>Keep changes when selecting a different transaction/split</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_EnterMovesBetweenFields">
         <property name="text">
          <string>Use Enter to move between fields</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_stringMatchFromStart">
         <property name="whatsThis">
          <string>Mark this option, if you always want to match names e.g. for payees from the start. If unset, any substring is matched.</string>
         </property>
         <property name="text">
          <string>Match names from start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_AutoReconciliation">
         <property name="whatsThis">
          <string>After entering the reconciliation data automatically detect the transactions that match that data (in some cases it might be not possible to do that).</string>
         </property>
         <property name="text">
          <string>Automatic reconciliation</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel2_1">
           <property name="text">
            <string>Initial cursor position within the date field</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="kcfg_initialDateFieldCursorPosition">
           <item>
            <property name="text">
             <string>Day</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Year</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="spacer16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>31</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel2_2">
           <property name="text">
            <string>Default reconciliation state</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KComboBox" name="kcfg_defaultReconciliationState">
           <property name="toolTip">
            <string>Default reconciliation state for transactions entered during reconciliation of an account</string>
           </property>
           <item>
            <property name="text">
             <string>Not reconciled</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item reconciliation status">Cleared</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reconciled</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="spacer15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>31</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="kcfg_AutoFillTransaction">
         <property name="title">
          <string>Autofill</string>
         </property>
         <property name="selectedId" stdset="0">
          <number>-1</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton1">
            <property name="whatsThis">
             <string>Do not auto-fill transaction data at all.</string>
            </property>
            <property name="text">
             <string>No Autofill</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton2">
            <property name="whatsThis">
             <string>Collect all transactions for the given payee. Treat all transactions that refer to the same category and have an amount with +/- X % as identical. If more than one transaction is found, a list of them is presented to the user.

Selecting 0% will list all transactions.</string>
            </property>
            <property name="text">
             <string>Same transaction if amount differs less than percentage set below</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton3">
            <property name="whatsThis">
             <string>The data of the last transaction assigned to the category used most often for this payee is autofilled into the transaction editor.</string>
            </property>
            <property name="text">
             <string>With previously most often used transaction for the payee</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="percentageWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Same transaction if amount differs less than </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="kcfg_AutoFillDifference">
            <property name="whatsThis">
             <string>Two transactions are usually treated identical for autofill, if they refer the same accounts. They are treated as different transactions though, when their amount varies by more than the percentage given here.</string>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_AutoFillUseMemos">
         <property name="whatsThis">
          <string>If this option is checked the memos from the previous transaction will be used otherwise the memos will not be considered when the transaction is autofilled.</string>
         </property>
         <property name="text">
          <string>Use memos from previous transaction</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage2">
      <attribute name="title">
       <string>Import</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel1_2">
           <property name="text">
            <string>Match transactions within days</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="kcfg_matchInterval">
           <property name="whatsThis">
            <string>Search for matching transactions within the range of the posting date of the imported transaction +/- the number of given days.</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>99</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_askForPayeeCategory">
         <property name="whatsThis">
          <string>Whenever a new payee is detected during import of a statement, the user will be asked to assign a default category for this user when this option is selected.</string>
         </property>
         <property name="text">
          <string>Ask for a new payee's default category</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer14">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TransactionSortOption</class>
   <extends>QWidget</extends>
   <header>transactionsortoption.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton2</sender>
   <signal>toggled(bool)</signal>
   <receiver>percentageWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
